tests/test_litellm/proxy/guardrails/guardrail_hooks/test_crowdstrike_aidr.py

```diff
@@ -0,0 +1,394 @@
+from unittest.mock import patch
+
+import httpx
+import pytest
+from fastapi import HTTPException
+
+from litellm.proxy.guardrails.guardrail_hooks.crowdstrike_aidr.crowdstrike_aidr import (
+    CrowdStrikeAIDRGuardrailMissingSecrets,
+    CrowdStrikeAIDRHandler,
+)
+from litellm.proxy.guardrails.init_guardrails import init_guardrails_v2
+from litellm.types.guardrails import GenericGuardrailAPIInputs
```

Comment:
GenericGuardrailAPIInputs was moved from litellm.types.guardrails to litellm.types.utils

```suggestion
from litellm.types.utils import GenericGuardrailAPIInputs
```

Reply:
Done.